docs/how-to/plan-upgrade.rst

```diff
@@ -23,72 +26,114 @@ Usage example
+                        Disable nova-compute scheduler from unit: 'nova-compute/0'
+                        Verify that unit 'nova-compute/0' has no VMs running
+                        ├── Pause the unit: 'nova-compute/0'
+                        ├── Upgrade the unit: 'nova-compute/0'
+                        ├── Resume the unit: 'nova-compute/0'
+                        Enable nova-compute scheduler from unit: 'nova-compute/0'
+                Wait for up to 1800s for model 'test_model' to reach the idle state
+                Verify that the workload of 'nova-compute' has been upgraded on units: nova-compute/0
+        Remaining Data Plane principal(s) upgrade plan
+            Upgrade plan for 'ceph-osd' to 'victoria'
+                Verify that all 'nova-compute' units had been upgraded
+                Upgrade software packages of 'ceph-osd' from the current APT repositories
+                    Upgrade software packages on unit 'ceph-osd/0'
+                Change charm config of 'ceph-osd' 'source' to 'cloud:focal-victoria'
+                Wait for up to 300s for app 'ceph-osd' to reach the idle state
+                Verify that the workload of 'ceph-osd' has been upgraded on units: ceph-osd/0
+        Data Plane subordinate(s) upgrade plan
+            Upgrade plan for 'ovn-chassis' to 'victoria'
+                Refresh 'ovn-chassis' to the latest revision of '22.03/stable'
+
+Plan for the control-plane
+--------------------------
+
+To generate a plan targeting the **control-plane** applications use:
+
+.. code:: bash
+
+    cou plan control-plane
+
+Plan for the data-plane
+-----------------------
+
+To generate a plan targeting the **data-plane** applications use:
+
+.. code:: bash
+
+    cou plan data-plane
+
+**Note:** It's essential to complete the upgrade of the **control-plane** components before
+being able to generate the plan for **data-plane**.
+
+
+Plan for the hypervisors
+------------------------
+
+To generate a plan targeting just the **hypervisors** use:
+
+.. code:: bash
+
+    cou plan hypervisors
+
+It's also possible to target for specific **availability-zones** or **machines**:
+
+.. code:: bash
+
+    # plan for just empty hypervisors in machines 0 and 1
+    cou plan hypervisors --machine "0, 1"
+
+    # plan for all empty hypervisors that are into zone-1
+    cou plan hypervisors --availability-zone=zone-1
+
+**Note:**:
+- Those specific filters are mutually exclusive, meaning that it's not possible
+to use then together.
+- Since **hypervisors** comprise a subset of **data-plane** components, it is
+also necessary to complete the upgrade of the **control-plane** components before
+being able to generate **hypervisors** plan.
+
+
+Plan for non-empty hypervisors
+------------------------------
+
+By default, COU will skip from the planning hypervisors that have VMs running. If it's
+necessary to also include then, use the `--force` command. For example:
```

Comment:
I think the default behavior should be mentioned in data-plane and hypervisors sections and point to this section about the usage of `--force`